src/stan-users-guide/regression.Rmd

```diff
@@ -25,24 +27,15 @@ $$
 y_n \sim \operatorname{normal}(\alpha + \beta X_n, \, \sigma).
 $$
 
-This latter form of the model is coded in Stan as follows.
+This latter form of the model is coded in Stan as follows [linear_regression_1](https://github.com/stan-dev/docs/tree/footnote/src/stan-users-guide/linear_regression_1)
 
-```
-data {
-  int<lower=0> N;
-  vector[N] x;
-  vector[N] y;
-}
-parameters {
-  real alpha;
-  real beta;
-  real<lower=0> sigma;
-}
-model {
-  y ~ normal(alpha + beta * x, sigma);
-}
-```
 
+```{r echo=FALSE}
```

Comment:
I'm still out on including files. If I were writing this, the fact that I can now link to a file makes it so that the only thing I'd put in the text itself would be a minimal code snippet:

```
model {
  y ~ normal(alpha + beta * x, sigma);
}
```

And then leave the full code example for the link.

Reply:
I am avoiding doing rewrite for now just to keep the size of the task manageable.  What you say makes sense.